scripts/babel-plugin-dynamic-to-static-import.js

```diff
@@ -0,0 +1,27 @@
+export default function(babel) {
+  const { types: t } = babel;
+
+  return {
+    visitor: {
+      ExpressionStatement(path) {
+        // find dynamic import expression
+        if (
+          t.isCallExpression(path.node.expression) &&
+          t.isImport(path.node.expression.callee)
+        ) {
+          // create static import
+          const staticImport = t.importDeclaration(
+            [],
+            t.stringLiteral(path.node.expression.arguments[0].value)
+          );
```

Comment:
this assumes the dynamic import will be in the form

```js
import("name")
```

which will transform to

```js
import "name";
```

it does **not** handle

```js
import("name").then((name) => /* do something */);
```

which would transform to

```js
import name from "name";
```

Reply:
resolved in c9f9feb